docs/source/release/index.rst

```diff
@@ -12,6 +12,8 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`release-pre-1.0`
 
+* :bug:`2829` Fix .drop(fields). The argument can now be either a list of strings or a
+string. In the latter case, the given string is transformed into one element list.
```

Comment:
If you check below, looks like for multi-line comments, we indent at the beginning of the test, not the bullet point (in short, can you indent this line two more spaces please).